docs/npwd/dev/FrameworkIntegration.md

```diff
@@ -0,0 +1,31 @@
+---
+id: framework_integration
+title: Framework Integration with NPWD
+sidebar_label: Framework Integration
+---
+
+NPWD was designed to be a framework agnostic resource that could work with any arbitary framework. To enable this,
```

Comment:
We also need to mention the camera event for photos as they will also need to work this into their own framework too. https://github.com/project-error/npwd/pull/403#issuecomment-986163547